app/models/stats/claim_stats.rb

```diff
@@ -0,0 +1,11 @@
+module Stats
+  class ClaimStats < Claim
+    scope :started, -> { where.not(password_digest: nil, state: 'submitted').within_timeframe }
+    scope :started_count, -> { started.count }
```

Comment:
Knowing that `scope` is just basically sugar for a class method I probably did this to be a little less verbose :mask: & also because its suffixed with count I thought the intention would be clear - not the case. I will change this, as it does break the expected return value of a `scope` which should always be a `Relation`. 

Thanks for pointing this out as although I did want to have a proper glance over this before assigning it I doubt I would have changed this.